netplan/cli/commands/apply.py

```diff
@@ -108,7 +108,7 @@ def command_apply(run_generate=True, sync=False, exit_on_error=True):  # pragma:
         # stop backends
         if restart_networkd:
             logging.debug('netplan generated networkd configuration changed, restarting networkd')
-            utils.systemctl_networkd('stop', sync=sync, extra_services=['netplan-wpa@*.service'])
+            utils.systemctl_networkd('stop', sync=sync, extra_services=['netplan-wpa-*.service'])
```

Comment:
i'd rather have `['netplan-wpa@*.service', 'netplan-wpa-*.service']`

After netplan upgrade, netplan generate will wipe old units, but they might still be running. Netplan apply then needs to stop them. It's ok to stop non-existent / unknown units. But maybe need to test that with older systemctls. If old systemctls doesn't handle the calls to stop non-existent / unknown units we then should do as two calls:
```
utils.systemctl_networkd('stop', sync=sync, extra_services=['netplan-wpa@*.service'])
utils.systemctl_networkd('stop', sync=sync, extra_services=['netplan-wpa-*.service'])
``

Reply:
Yes, I'd rather go the safe way, using two calls.
This way we can also better separate the two calls with a comment about this scenario.